awx/api/views/bulk.py

```diff
@@ -0,0 +1,51 @@
+from collections import OrderedDict
+
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.renderers import JSONRenderer
+from rest_framework.reverse import reverse
+from rest_framework import status
+from rest_framework.response import Response
+
+from awx.api.generics import (
+    GenericAPIView,
+    APIView,
+)
+from awx.api import (
+    serializers,
+    renderers,
+)
+
+
+class BulkJobLaunchView(GenericAPIView):
```

Comment:
@jainnikhil30 inheriting from `GenericAPIView` allows us to inherit some magic that renders the OPTIONS based on the `serializer_class` and `allowed_methods` for this view